.travis.yml

```diff
@@ -57,7 +57,7 @@ install:
   - yarn install --force
 
 script:
-  - yarn lint && yarn build:prod && yarn test:setup && yarn test
+  - yarn lint && yarn build:prod && yarn validate-changelog && yarn test:setup && yarn test
```

Comment:
I think we should validate the `change-log` first since it completes a lot faster than building. Nothing worse than waiting on CI for 15 mins just to find out our change-log is messed up.

Reply:
I want to clarify that I meant we should validate before building.